packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/accept-all/accept-all.routes.js

```diff
@@ -0,0 +1,46 @@
+import head from 'lodash/head';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing.autorenew.agreements.accept-all', {
+    url: '/accept-all',
+    views: {
+      modal: {
+        component: 'billingAutorenewActivation',
+      },
+    },
+    params: {
+      agreements: null,
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
+    resolve: {
+      activateAutorenew: /* @ngInject */ (BillingAutoRenew, nicRenew) => () =>
+        BillingAutoRenew.enableAutorenew(head(nicRenew.renewDays)),
+      agreements: /* @ngInject */ ($transition$) =>
+        $transition$.params().agreements,
+      goBack: /* @ngInject */ ($state, $timeout, Alerter) => (
+        reload = false,
+        message = false,
+        type = 'success',
```

Comment:
Imo, `message` parameter should be the first one because it has more chance of being defined than the reload and type one, therefore we will not be obliged to always pass a value for `reload` each time we want to give a custom message

Reply:
Got it. This is fixed